cli_client/python/timesketch_cli_client/commands/sketch.py

```diff
@@ -53,3 +55,43 @@ def create_sketch(ctx, name, description):
         description = name
     sketch = api_client.create_sketch(name=name, description=description)
     click.echo(f"Sketch created: {sketch.name}")
+
+
+@sketch_group.command("add_event")
```

Comment:
This should be using kebab case, `add-event` to be consistent with the rest of the CLI UI